packages/umi/routerTypes.d.ts

```diff
@@ -8,8 +8,8 @@ type IncludeRoute = 'component' | 'exact' | 'path';
 
 type RouteType = Pick<RouteProps, IncludeRoute>;
 
-export default interface RouterTypes<T extends Object = {}>
+export default interface RouterTypes<T extends Object = {}, P extends { [K in keyof P]?: string } = {}>
```

Comment:
```suggestion
export default interface RouterTypes<T extends Object = {}, P = {}>
```

match 里有 `[K in keyof Params]?: string`